### DISCUSSION TITLE
WebProxy Https

### DISCUSSION THREAD

**Discussion by jkvargas** [dotnet/runtime#64349] [Q&A]
As I understand https with WebProxy is not supported.

something on the lines of

```
 .ConfigureServices(services =>
    {
        services.AddHttpClient()
            .ConfigurePrimaryHttpMessageHandler(() => new HttpClientHandler
            {
                Proxy = new WebProxy
                {
                    Address = new Uri(httpsAddress),
                },
...
```

"Unhandled exception. System.NotSupportedException: Only the 'http', 'socks4', 'socks4a' and 'socks5' schemes are allowed for proxies."

is there a workaround or another solution?

**Comment by huoyaoyuan** [comment]
The proxy support ties closely to http connection handler. Currently there's no support for https proxy. Adding support is non-trivial.
